modules/rational-compile.el

```diff
@@ -21,28 +21,59 @@
 
 ;;; Code:
 
+
 ;;;; Variables:
-;; We define a variable to enable/disable modules autocompilation.
+;; We define a variable to enable/disable modules compilation.
 (defvar rational-compile-modules t
-  "When non-nil, autocompile emacs lisp sources for the modules on startup.")
+  "When non-nil, compile emacs lisp sources for the modules on startup.")
+
+;; We define a variable to enable/disable user's modules compilation.
+(defvar rational-compile-user-modules t
+  "When non-nil, compile user's emacs lisp sources for the modules on startup.")
+
+(defvar rational-compile-module-list '()
+  "A list of modules to compile.")
+
+(defvar rational-compile-modules-path
```

Comment:
This var may not be needed if the suggested change to the locate modules code is adopted (ie use the call to `rational-compile--config-dirs-list` instead).

Reply:
This variable must be used, since is the way the user will define which modules will be compiled. If we only compile all the sources in the directories that are in the path, the issue #144 will not be solved.

The issue #144 is that this module will try to compile **all** the modules, even the ones not in use, which will cause dependency problems if all the dependencies are not installed.

This variable ensures that only the modules that the user want to compile will be compiled.

Reply:
Ah, I guess I misunderstood the code then, it appeared this was the path(s) to the modules and was only used in 2 places. Thanks for the clarification.